aws/wafv2_helper.go

```diff
@@ -382,6 +382,21 @@ func wafv2TextTransformationSchema() *schema.Schema {
 						wafv2.TextTransformationTypeLowercase,
 						wafv2.TextTransformationTypeNone,
 						wafv2.TextTransformationTypeUrlDecode,
+						wafv2.TextTransformationTypeBase64Decode,
```

Comment:
As described in (#14601) the `aws-go-sdk` has added `_Values()` suffixed const arrays so we can actually replace this whole statement with:

```
validation.StringInSlice(wafv2.TextTransformationType_Values(), false),
```
This way the validation will automatically update as we update `aws-go-sdk` versions.

Reply:
Wow! Thanks for this tidbit of info, super slick. Will get this updated shortly!